### DISCUSSION TITLE
Workshop

### DISCUSSION THREAD

**Discussion by dietmarwo** [esa/SpOC3#17] [Competition Q&A]
For SpOC 1 ansd 2 we had a workshop discussing the solution approaches (see https://indico.esa.int/event/486/ ). 
Will there be a workshop for SpOC 3? Or some other way we can discuss solution approaches?

**Comment by maxbannach** [comment]
See https://github.com/esa/SpOC3/discussions/16

**Reply by dietmarwo** [reply]
May be it would have been a good idea to reach out to the participants via mail, as it was done in previous years. As far as I know neither the Honda team nor fcmaes were aware about that virtual workshop.

**Reply by maxbannach** [reply]
Yeah, true. I thought a notification would be sent. Anyhow, we will adapt the procedure for subsequent iterations.
